### PR TITLE
Create 0009-Archivematica-original-design-principles.md

### PR DESCRIPTION
**Pull request by evelynPM**
This is an ADR for the original design principles underpinning Archivematica

Rendered markdown: [here](https://github.com/archivematica/archivematica-architectural-decisions/blob/63a250f8667d64bee8363a63d466e96c84408011/0009-Archivematica-original-design-principles.md).

Connected to https://github.com/archivematica/archivematica-architectural-decisions/issues/21

### CODE REVIEW

File: 0009-Archivematica-original-design-principles.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+* https://wiki.archivematica.org/Overview
+* https://wiki.archivematica.org/Dashboard Original dashboard functional requirements
+* https://wiki.archivematica.org/AIP_structure Reason for quad directories
+* https://smartech.gatech.edu/handle/1853/28490 “Permanent Objects, Disposable Systems”, Abrams, Stephen; Cruse, Patricia; Kunze, John, 2009
+* https://escholarship.org/uc/item/5313h6k9 “An emergent micro-services approach to digital curation infrastructure”, Abrams, Stephen; Kunze, John; Loy, David, 2009
+* Artefactual ipres presentations:
+* ipres2010: https://drive.google.com/file/d/0Bw6XXtMa5JRpcHp1Q0ZSV0tqdU0/view?usp=sharing 
+* ipres2012: https://drive.google.com/open?id=0B2qNaVLxjZmKamJkVmVFd3l1cVE 
+* ipres2013: https://drive.google.com/open?id=0B5OYDF_LHu4RY0ViLUlOd0hKUDA
```
Comment (ross-spencer) [inline 0009-Archivematica-original-design-principles.md]: I had a look at https://phaidra.univie.ac.at/search#?page=1&pagesize=10&collection=o:340041 and it seems that only a select few IPRES proceedings have been broken down into individual papers. An alternative to hosting on GDrive that might be more sustainable in the future is to use our https://github.com/archivematica/archivematica-case-studies resources folder, what do you think?

File: 0009-Archivematica-original-design-principles.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Decision drivers
+
+Artefactual designed an open-source, web-based archival description and access system called ICA-AtoM (Access To Memory) that has a broad user base around the world. ICA-AtoM does not provide digital preservation functionality as described by OAIS. It would benefit ICA-AtoM users to be able to integrate with a back-end system designed to preserve digital objects that are linked to access copies in ICA-AtoM. The system should also be usable on its own or in conjunction with other access tools.
```
Comment (ross-spencer) [inline 0009-Archivematica-original-design-principles.md]: The broader use of Archivematica came up in a meeting earlier and it made me wonder if perhaps the problem statement could be expanded to describe its interoperability with a wide range (read: any) archival management system/content management system, etc.

**Comment by corinnemrogers** [comment]
This is super interesting to read - I think it would be helpful to add some dates.

**Comment by peterVG** [comment]
Is there a reason this ADR has not been approved yet? Numbers 0009 and 0000 are taken so it would have to be renumbered to 0010.

**Comment by peterVG** [comment]
Actually, the ADR content says "status: accepted" but it is not listed in the Accepted log: https://github.com/archivematica/archivematica-architectural-decisions/blob/master/accepted-log.md and it hasn't been merged to the Master branch?

**Comment by ross-spencer** [comment]
@peterVG I don't think it has been through its first round of review proper yet. I think it was simply marked as accepted from the outset :)

File: 0009-Archivematica-original-design-principles.md (hunk 1 above)
Comment (ross-spencer) [inline 0009-Archivematica-original-design-principles.md:49]:
```suggestion
* iPres 2010: "Archivematica: Using Micro-services and open-source software to deliver a comprehensive digital curation solution." [article](https://github.com/archivematica/archivematica-case-studies/blob/26a7a9f6ae827ae69955c6190b4106fe8c33ee5a/resources/2010-09-iPres-Archivematica-UsingMicroServicesAndOpenSourceSoftware-VanGarderen.pdf).
* iPres 2012: "The Community­driven Evolution of the Archivematica Project." [article](https://github.com/archivematica/archivematica-case-studies/blob/26a7a9f6ae827ae69955c6190b4106fe8c33ee5a/resources/2012-10-iPres-Paper49-Community-driven-evolution-of-Archivematica-project-VanGarderen-Mumma.pdf).
* iPres 2013: "Realizing the Archivematica vision: delivering a comprehensive and free OAIS implementation." [article](https://github.com/archivematica/archivematica-case-studies/blob/26a7a9f6ae827ae69955c6190b4106fe8c33ee5a/resources/2013-09-02-Realizing-the-Arhivematica-Vision-MummaVanGarderen-iPres.pdf).
```

**Comment by sromkey** [comment]
> Is there a reason this ADR has not been approved yet? It would be nice to be able to refer to it in upcoming ADRs. Note that ADR numbers 0009 and 0000 are taken so it would have to be renumbered to 0010. See #23

Apologies- no reason whatsoever other than being busy and it not being a particularly pressing need. I suggest filing new ADRs with the next available number and this one can be adjusted when it's time to merge.

**Comment by peterVG** [comment]
friendly bump to please review this PR

**Review by sevein — APPROVED** [review]

File: 0010-Archivematica-original-design-principles.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Artefactual designed an open-source, web-based archival description and access system called ICA-AtoM (Access To Memory) that has a broad user base around the world. ICA-AtoM does not provide digital preservation functionality as described by OAIS. It would benefit ICA-AtoM users to be able to integrate with a back-end system designed to preserve digital objects that are linked to access copies in ICA-AtoM. The system should also be usable on its own or in conjunction with other access tools.
+
+## Considered options
+
+1. Develop a repository interface to a digital object store (e.g. Fedora or DSpace); or
+2. Develop a database-backed application to manage AIPs (e.g. Preservica); or
+3. Use a micro-services approach with loosely-coupled tools to provide digital curation services built around file system storage (e.g. University of California’s Merritt System).
+
+
```
Comment (sevein) [inline 0010-Archivematica-original-design-principles.md:24]: Peter, the linter is reporting some minor issues like this extra line being reported as:

    MD012/no-multiple-blanks Multiple

It's not a big deal though, we can fix some other time.

Comment (peterVG) [reply 0010-Archivematica-original-design-principles.md:24]: thanks

**Pull request merged**